support/fixtures/factories/contxtUserPermissions.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+const factory = require('rosie').Factory;
+const faker = require('faker');
+const times = require('lodash.times');
+
+factory
+  .define('contxtUserPermissions')
+  .option('fromServer', false)
+  .attrs({
+    applicationsExplicit: () =>
+      times(
```

Comment:
Fun fact I learned a couple weeks ago: You can do this with Array.from!
```
Array.from(
  { length: faker.random.number({ min: 0, max: 15}) },
  () => factory.build('contxtApplication').id
)
```

Not a request! Repeat! Not a request: Do you like it better this way? I can't decide
```
factory
  .buildList('contxtApplication', faker.random.number({ min: 0, max: 15}))
  .map(({ id }) => id)
```
```

Reply:
I was not aware of `Array.from()` thanks for sharing.

And yes, I prefer your way better. Keep's lodash out of it :+1: